husarion_ugv_crsf_teleop/husarion_ugv_crsf_teleop/crsf_teleop_node.py

```diff
@@ -0,0 +1,187 @@
+        
+        self.declare_parameter('port', '/dev/ttyUSB0', ParameterDescriptor(description='CRSF receiver serial port'))
+        self.declare_parameter('baud', 576000, ParameterDescriptor(description='CRSF receiver baud rate'))
+        self.declare_parameter('e_stop_republish', False,
+                               ParameterDescriptor(description='Rebroadcast asserted e-stop signal once per second'))
+        self.declare_parameter('enable_cmd_vel_silence_switch', False,
+                               ParameterDescriptor(description='Enable cmd_vel silence switch allowing other nodes to take control'))
+        self.declare_parameter('linear_speed_presets', [0.5, 1.0, 2.0], ParameterDescriptor(description='Selectable robot maximum linear speed for cmd_vel', floating_point_range=[FloatingPointRange(from_value=0.0, to_value=10.0)]))
+        self.declare_parameter('angular_speed_presets', [0.5, 1.0, 2.0], ParameterDescriptor(description='Selectable robot maximum angular speed for cmd_vel', floating_point_range=[FloatingPointRange(from_value=0.0, to_value=10.0)]))
+        self.linear_speed_presets = self.get_parameter('linear_speed_presets').get_parameter_value()._double_array_value
+        self.angular_speed_presets = self.get_parameter('angular_speed_presets').get_parameter_value()._double_array_value
+        
+        if len(self.linear_speed_presets) != 3 or len(self.angular_speed_presets) != 3:
+            raise ValueError("Speed presets must be a list of 3 values")
+        
+        # Initialize serial port
+        port = self.get_parameter('port').get_parameter_value().string_value
+        baud = self.get_parameter('baud').get_parameter_value().integer_value
+        
+        self.serial = serial.Serial(port, baud, timeout=2)
+        
+        # Setup CRSF parser
+        self.parser = CRSFParser()
+        self.parser.on_message = lambda msg: self.handle_message(msg)
+        
+        self.timer = self.create_timer(0.01, self.run)
+        
+        self.rc_estop_state = True
+        if self.get_parameter('e_stop_republish').get_parameter_value().bool_value:
+            self.e_stop_republisher = self.create_timer(1, self.update_e_stop)
+        
+    def run(self):
+        if self.serial.in_waiting > 0:
+            self.parser.parse(self.serial.read(self.serial.in_waiting))
+                
+    def handle_message(self, msg: CRSFMessage):
+        if msg.msg_type == PacketType.RC_CHANNELS_PACKED:
+            channels = unpack_channels(msg.payload)
+            
+            # Map channels from CRSF range [172, 1812] to [-1, 1]
+            channels = [(channel - 992)/ 820.0 for channel in channels]
+            
+            # Handle emergency stop from RC controller
+            # Asserted e-stop is retransmitted once per second by republish timer
+            # if enabled by the 'e_stop_republish' parameter
+            # Deasserted e-stop is transmitted only once
+            requested_e_stop = channels[4] < 0.5
```

Comment:
Please add enums for with names of channels.
Proposition:
```suggestion
            requested_e_stop = channels[switch.SF] < 0.5
```